### DISCUSSION TITLE
Feature Request: FontAwesomeIcons in NavbarMinimal

### DISCUSSION THREAD

**Discussion by PhilBrk8** [mantinedev/mantine#1798] [Q&A]
Hi you guys, 
first things first: So I'm from a DataScience background and am pretty new to all of this, so thank you for this awesome Library. I´d never be able to do this on my own.
secondly: my pythonic lifestyle is also pretty new to this weird statically typing thing, so maybe what I want to achieve even is possible with little to no effort...

anyways, here is my problem:
I'd like to use fontawesome icons in the navbar and not just react-tabler-icons.
I integrated the NavbarMinimal into the Apshell and would like to be able to just include an imported fontawesome- icon in the given structure like this:
![image](https://user-images.githubusercontent.com/92030164/179342978-8fbbda3b-4657-40dd-8c91-7a4a90c37356.png)
![image](https://user-images.githubusercontent.com/92030164/179342964-53c13e23-a9d4-46d6-a8a6-89c42c737ff3.png)

The following errors are thrown, but frankly... I have no Idea what I'm doing there. - Help is highly appreciated, thanks.
![image](https://user-images.githubusercontent.com/92030164/179343148-1d6949bb-5eba-46b3-aee3-6cb017d9800f.png)
![image](https://user-images.githubusercontent.com/92030164/179343245-f2023445-6cec-4de4-8587-ef6a1b40e63d.png)

**Comment by rtivital** [comment]
Hi @PhilBrk8, thanks for kind words! Mantine UI provides examples of how Mantine components can be used to build other, more complex components – we cannot provide an example for integration of each third-party package, you will need to figure that out on your side.

**Reply by PhilBrk8** [reply]
hoped for a plug and play solution, but kind of expected what you answered.
Thanks for taking the time to reply anyways
